README.md

```diff
@@ -1,2 +1,92 @@
-# RFCs
-Holds RFCS (“Request for Comments”), proposals to improve Enarx
+# RFCs for Enarx
+
+This repo holds Request for Comments (RFCs), enhancement proposals to improve
+the Enarx project.
+
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the
+[normal GitHub pull request workflow](https://github.com/enarx/enarx/wiki/How-to-contribute).
+
+Some changes though are "substantial", and we ask that these be put through a
+bit of a design process and produce a consensus among the community and
+the [sub-team]s.
+
+If you are here to learn about Enarx, as the code is still young, we
+recommend you look at the [code][https://github.com/enarx/enarx/) and
+[wiki](https://github.com/enarx/enarx/wiki) in the
+[main Enarx repo](https://github.com/enarx/enarx/).
+
+In the future, you will be able to browse [the RFC Index](index.md) for
+a current listing of all RFCs and their statuses.
+
+There are 2 types of Enarx RFCs:
+
+* RFCs that describe individual features (in the [features](./features) folder)
+* RFCs that explain concepts underpinning many features
+(in the [concepts](./concepts) folder)
+
+## RFC Lifecycle
+
+RFCs go through a standard lifecycle.
+
+![lifecycle](lifecycle.png)
+
+#### PROPOSED
+To __propose__ an RFC, [use these instructions to raise a PR](
+contributing.md#how-to-propose-an-RFC) (Pull Request) against the repo.
```

Comment:
~~use these instructions to raise a PR (Pull Request)~~ -> use these instructions to raise a Pull Request (PR)